spec/support/schema_validation.rb

```diff
@@ -106,7 +118,9 @@ def self.validate_record!(record, schema)
       validate_field!(record, key, fs, value)
     end
     # 2) All non-optional fields in schema are found in record
-    # ABS FIXME
```

Comment:
To think, I'd even left myself a note to come back and fix this, then forgot all about it - and didn't notice it in review :'-(